docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_pandas_filesystem_datasource.rst

```diff
@@ -222,7 +234,7 @@ Additional Notes
 ----------------
 
 #.
-    For the Stable API (up to 0.12.x), relative path locations should be specified from the perspective of the directory, in which the
+    For the Stable API (up to 0.12.x), relative path locations (e.g. for the ``base_directory``) should be specified from the perspective of the directory, in which the
```

Comment:
@jcampbell Could you please clarify your question?  I can see that @spbail added the parenthetical remark, which do not see an issue with.  Thank you.

Reply:
The sentence is literally cut off, so I don't know what it says. For me, it reads:

"For the Stable API (up to 0.12.x), relative path locations (e.g. for the ``base_directory``) should be specified from the perspective of the directory, in which the" ... and I have no idea which directory it's talking about.

What actually changed here between the APIs?

Reply:
Oh duh, I can change that. Nothing changed between the APIs, but the CLI and yaml behavior are different (since the CLI is run from the parent directory of GE), which is made explicit here.

Reply:
@jcampbell You are correct; the sentence was cut off.  I will figure out with @spbail what happened there.

Reply:
@jcampbell Looks like this problem has existed in `develop` -- I will fix it now in this PR. /cc @spbail

Reply:
That's just github formatting cutting it off here, the actual page is correct. If you look at the full page, the sentence contains a code block:

For the Stable API (up to 0.12.x), relative path locations (e.g. for the base_directory) should be specified from the perspective of the directory, in which the

great_expectations datasource new
command is executed.